var/spack/repos/builtin/packages/neovim/package.py

```diff
@@ -4,37 +4,63 @@
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
 
+import inspect
+import os
```

Comment:
I think only `os.path` is used, and for `inspect` see ~below~ above:
```suggestion
import os.path
```